doc/source/parameters/cfd/multiphysics.rst

```diff
@@ -38,7 +39,15 @@ This subsection defines the multiphysics interface of Lethe and enables the solu
 * ``VOF``: enables multiphase flow simulations, with two fluids separated by a free surface, using the Volume-of-Fluid method. 
 
   See :doc:`volume_of_fluid` for advanced VOF parameters, :doc:`initial_conditions` for the definition of the VOF conditions and `Physical properties - two phase simulations <https://lethe-cfd.github.io/lethe/parameters/cfd/physical_properties.html#two-phase-simulations>`_ for the definition of the physical properties of both fluids.
-
+  
+* ``use average velocity field``: controls if the advection of the subphysics is done using the average velocity field instead of the current velocity field. This is useful when the flow dynamics and the subphysics reach a pseudo-steady state at different time scales.
```

Comment:
Also, I think the parameter could have a more meaningful name. My suggestions would be "use time averaged velocity field", "use steady state velocity field" or "use constant velocity field". The "average" gives an idea of spatial average instead of temporal average.

Reply:
Agreed. Use time-averaged would be the best way to write it.